### DISCUSSION TITLE
Friends new theme: FediPress

### DISCUSSION THREAD

**Discussion by liviacarolgouvea** [akirk/friends#425] [Show and tell]
Hi Alex!!

We created a new theme for Friends: FediPress (https://github.com/alquimidia/fedipress/releases/tag/v1.0.0)

We are waiting Wordpress.org to reopen submissions for new plugins to submit there.

If possible, we would like to know your opinion! :)

Hugs!

**Comment by akirk** [comment]
This is great, thank you!! Adding it to the plugins section in https://github.com/akirk/friends/pull/433
